lib/events/usageevents/usageevents.go

```diff
@@ -67,10 +67,30 @@ func (u *UsageLogger) reportAuditEvent(ctx context.Context, event apievents.Audi
 			ConnectorType: e.Method,
 		}))
 	case *apievents.SessionStart:
-		// Note: session.start is only SSH.
+		// Note: session.start is only SSH and Kubernetes.
+		sessionType := types.SSHSessionKind
```

Comment:
This should be `ssh_v2` or something along those lines, as we can't really trust `ssh` to mean SSH sessions now. 😢

Either that, or we should have a `UsageSessionStartV2` event (same underlying type, just a different oneof entry).

Reply:
That's reasonable, I've added a new enum variant for V2 and switched everything over to it.